sdks/java/io/hbase/pom.xml

```diff
@@ -52,29 +50,6 @@
               </goals>
               <configuration>
                 <createDependencyReducedPom>false</createDependencyReducedPom>
```

Comment:
Thinking a little bit about it I do have to shade it, because it will leak into the user classpath mmm... not so happy about this, I don't know how we would deal with this for the case of GCP IOs that are not shaded and will leak too.